docs/src/main/sphinx/release/release-369.md

```diff
@@ -0,0 +1,104 @@
+# Release 369 (19 Jan 2022)
+
+## General
+
+* Display `Physical input read time` using most succinct time unit in `EXPLAIN
+  ANALYZE VERBOSE`. ({issue}`10576`)
+* Fine tune request retry mechanism in HTTP event listener. ({issue}`10566`)
+* Remove unnecessary fine-grained spill toggles. ({issue}`10591`)
+* Prevent hanging query execution on failures with `phased` execution policy.
+  ({issue}`10656`)
+* Fix `UnsupportedOperationException` in `max_by` and `min_by` aggregation.
+  ({issue}`10599 `)
+* Fix error message for function calls with lambda arguments. ({issue}`10423 `)
```

Comment:
@mosabua Could you share why you decided to include this entry at https://github.com/trinodb/trino/issues/10552#issuecomment-1014770155? I'm asking this question because we didn't include the change for error message in the past release notes as far as I know.

Reply:
I wasnt aware of that policy. I can remove but honestly I think these kinds of things should be added. Up to you and @electrum

Reply:
wdyt @electrum ?

Reply:
Removed as discussed with @martint